index.html

```diff
@@ -1,69 +1,69 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ko">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/variables.css">
+    <link rel="stylesheet" href="css/index/style.css">
     <title>판다마켓</title>
 </head>
 <body>
     <header>
         <nav class="herader-nav">
-                <a class="header__logo-box": href ="/">
-                    <img class="header__logo" src="images/logo/logo.png">
-                    <span class="header__title">판다마켓</span>
-                </a>
-
+            <a class="header__logo-box" href ="/">
+                <img class="header__logo" src="images/logo/logo.png" alt="판다 마켓 메인으로 이동">
+            </a>
```

Comment:
💊 제안
heading 태그의 mdn 정의를 보면, 가능하면 h1태그의 사용을 권고하고 있습니다~
이는 접근성을 위한 권고사항이라 필수는 아니지만 인성님께서 접근성에 관심이 있으신 것 같아 첨언드리자면
일반적으로 한 페이지에는 한 번만 사용되는 h1 태그를 로고를 감싸는 데 활용하는 경우가 많습니다.
따라서 아래처럼 하시는 것을 추천드릴께요~

[mdn](https://developer.mozilla.org/ko/docs/Web/HTML/Element/Heading_Elements#%ED%8A%B9%EC%84%B1)

```suggestion
            <h1>
                <a class="header__logo-box" href ="/">
                    <img class="header__logo" src="images/logo/logo.png" alt="판다 마켓 메인으로 이동">
                </a>
            </h1>
```